### PR TITLE
Add DefaultReader from aicsimageio

### PR DESCRIPTION
**Pull request by SeanLeRoy**
Initialize this repository (beyond what the cookiecutter supplies). This was previously `DefaultReader` in `aicsimageio`

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1 @@
+recursive-include scripts *.txt
```
Comment (evamaxfield) [inline MANIFEST.in:1]: this should be moved to the `pyproject.toml` file. Here: https://github.com/bioio-devs/bioio-imageio/blob/main/pyproject.toml#L77

File: bioio_imageio/metadata_utils.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
+from typing import Union
+
+
+def generate_ome_image_id(image_id: Union[str, int]) -> str:
```
Comment (evamaxfield) [inline bioio_imageio/metadata_utils.py:4]: how many repos has this been copied and pasted to? sort of tempted to say this should live in `bioio_base`

File: bioio_imageio/reader_metadata.py [hunk 3]
```diff
@@ -21,8 +21,7 @@ def get_supported_extensions() -> List[str]:
         """
         Return a list of file extensions this plugin supports reading.
         """
-        raise NotImplementedError()
-        # return ["ext", "extn"]
+        return ["bmp", "gif", "jpg", "mp4", "png"]
```
Comment (evamaxfield) [inline bioio_imageio/reader_metadata.py:24]: not sure if we want to include more but here is the full list: https://imageio.readthedocs.io/en/stable/formats/index.html

**Review by evamaxfield — APPROVED** [review]
looks good! Some minor recommendations / questions!

File: bioio_imageio/metadata_utils.py (hunk 2 above)
Comment (SeanLeRoy) [reply bioio_imageio/metadata_utils.py:4]: I've been wondering this too, might be a good move for the next `bioio_base` version

**Review by BrianWhitneyAI — APPROVED** [review]
Nice!

**Pull request merged**